src/components/ComponentPages/CreateNewTopic/TopicUI.tsx

```diff
@@ -37,116 +36,111 @@ const CreateNewTopicUI = ({
         <SideBar />
       </aside>
       <div className="pageContentWrap">
-        <Spinner>
-          <Card title={CardTitle} className="can-card-style">
-            <Form
-              form={form}
-              onFinish={onFinish}
-              name="create_new_topic"
-              className={`${styles.createNewTopicForm}`}
-              layout={"vertical"}
-              autoComplete="off"
-              scrollToFirstError
-              validateTrigger={messages.formValidationTypes()}
-              initialValues={{
-                ...initialValue,
-                namespace: nameSpaces && nameSpaces[0]?.id,
-              }}
-            >
-              <Row gutter={16}>
-                <Col xs={24} sm={12}>
-                  {nickNameList.length ? (
-                    <Form.Item
-                      label={labels.cr_nick_name}
-                      name="nick_name"
-                      {...nickNmRule}
-                      extra={labels.cr_nick_name_sp}
-                      initialValue={nickNameList[0]?.id}
-                    >
-                      <Select
-                        placeholder={placeholders.nickName}
-                        allowClear
-                        size={"large"}
-                        defaultValue={nickNameList[0]?.id}
-                      >
-                        {nickNameList.map((nick, idx) => (
-                          <Option key={nick.id} value={nick.id}>
-                            {nick.nick_name}
-                          </Option>
-                        ))}
-                      </Select>
-                    </Form.Item>
-                  ) : null}
-
+        <Card title={CardTitle} className="can-card-style">
+          <Form
```

Comment:
Please move form in a separate components

Reply:
Done.